BrainPortal/db/schema.rb

```diff
@@ -41,6 +41,11 @@
 
   add_index "active_record_logs", ["ar_id", "ar_table_name"], :name => "index_active_record_logs_on_ar_id_and_ar_table_name"
 
+  create_table "bourreaux_tools", :id => false, :force => true do |t|
```

Comment:
That table is spurious! Where does it even come from?!? There's no many-to many assoc between bourreaux and tools!